application/src/logmanager.py

```diff
@@ -41,18 +77,22 @@ def output_breathing_log(self, number_of_lines):
         print("\nCurrent Breathing Session_______________________________________")
         print(pretty_last_n_current_time)
 
-    def convert_time_entry_to_pretty_output(self, time_entry_line):
+    def convert_entry_to_pretty_output(self, time_entry_line):
         # Convert to an array
         values = time_entry_line.split('\t')
-        # Get the time. Pretty format it.
+
+        # Get the datetime. Pretty format it.
         entry_time = values[0]
         time_obj = datetime.datetime.strptime(entry_time, '%Y-%m-%d %H:%M:%S.%f')
         time_string = time_obj.strftime('%b %d %Y -- %H:%M')
 
-        # For each of the times, pretty format it into seconds and minutes
-        breathing_seconds_from_log = list(map(int, values[1:]))
+        # Get the journal entry
+        journal_entry = values[1]
```

Comment:
Can we do a check to avoid a backward-compatible breaking change?

Reply:
What kind of check do you mean here?

Reply:
As soon as anyone loads this code, the values[1] will be a time and not a journal entry; right?

Reply:
Hotfixed file name change.